config/cmd/configure.go

```diff
@@ -41,6 +42,7 @@ type DefaultsData struct {
 	InitImagePrefix  string
 	ImagePrefix      string
 	BuildImagePrefix string
+	KillAfter        float64
```

Comment:
nit: it seems like killafter should be an integer?

Reply:
I think it is a minor issue whether it is one or the other, but the kill-after flag in timeout is a float, so there is no harm in allowing it to be specified as a float here.